inspirehep/utils/bibtex.py

```diff
@@ -318,7 +318,7 @@ def _get_collaboration(self):
         collaboration = ''
         if 'collaboration' in self.record:
             try:
-                collaboration = self.record['collaboration'][0]
+                collaboration = self.record['collaboration'][0]['value']
             except IndexError:
```

Comment:
So `self.record['collaboration.value']` will access the first element of the list?